src/vsr/replica.zig

```diff
@@ -682,19 +681,23 @@ pub fn ReplicaType(
         }
 
         fn free_set_open_callback(free_set_encoded: *FreeSetEncoded) void {
-            const superblock = @fieldParentPtr(SuperBlock, "free_set_encoded", free_set_encoded);
-            const self = @fieldParentPtr(Self, "superblock", superblock);
+            const grid = @fieldParentPtr(Grid, "free_set_encoded", free_set_encoded);
+            const self = @fieldParentPtr(Self, "grid", grid);
             assert(!self.state_machine_opened);
             assert(self.commit_stage == .idle);
             assert(self.syncing == .idle);
             assert(self.sync_tables == null);
             assert(self.grid_repair_tables.executing() == 0);
```

Comment:
```suggestion
            assert(self.grid_repair_tables.executing() == 0);
            assert(self.grid.free_set.count_released() == self.grid.free_set_encoded.block_count);
```

This serves as a nice hint that our state here matches the state immediately after the same checkpoint.